release.nix

```
@@ -6,6 +6,11 @@
     # Find the current version number with `git log --pretty=%h | wc -l`
     entries = [
       {
+        version = 570;
+        changes = ''
+          Made the event stream Varlink events public, with sum-style types.
```

```suggestion
          Make the event stream Varlink events public, with sum-style types.
```

For consistency - the release notes are written in the present tense.